src/app/views/panels/scale_value_selector.tsx

```diff
@@ -136,7 +136,8 @@ export class ScaleValueSelector extends React.Component<
                   onClick={() => {
                     new Actions.ToggleLegendForScale(
                       scale._id,
-                      scaleMapping
+                      scaleMapping,
+                      null // TODO find the plot segment
```

Comment:
Fix this

Reply:
@ilfat, I may have [a fix for this](https://github.com/microsoft/charticulator/commit/8c6fe1a77babf53bd83e04b23ba251640aca90b3), but I am unsure of how this code is invoked.

Reply:
Add checks PS for null in toggleLegendForScale and pass null here